### DISCUSSION TITLE
添加翻译内容的顺序

### DISCUSSION THREAD

**Discussion by versun** [immersive-translate/immersive-translate#1343] [Ideas]
### 新功能 | New Feature

目前：在翻译全文时，默认是翻译主要区域/全部区域
新功能：在翻译全文时，第一次Alt-A可设置为先翻译标题(所有h1)，第二次Alt-A再翻译子标题(所有h2)，第三次Alt-A然后再翻译加粗的字体，第四次Alt-A翻译主要区域/全部区域

### 为什么您想要添加这些功能 | Why you want to add these features

1. 节省翻译量
2. 大部分时间，翻译全文后只是为了快速浏览标题，如果感兴趣才会再浏览具体内容
3. 虽然可以单独点击翻译，但效率慢

### 提议 | Proposal

该功能可做为附加选项，默认还是维持原来的翻译主要区域/全部区域，如需更细致的翻译则可以自行设置

**Comment by theowenyoung** [comment]
这样逻辑好像就有点复杂了..

(我建议别使用 openai 翻译 就好了.. 其他翻译服务实际上很快？

**Reply by versun** [reply]
嗯嗯，只是一个突发奇想的idea
